### PR TITLE
Use log package from stdlib and some housekeeping

### PR DESCRIPTION
**Pull request by blockchainjamie**
As requested, another PR for housekeeping changes 👍

### CODE REVIEW

File: src/github.com/eugenmayer/concourse-static-resource/in/in_command.go [hunk 1]
```diff
@@ -35,8 +35,8 @@ func main() {
 	// though when having a out to in handover, it would be the version out had, maybe something else VersionFromFile
 	var version string = request.Version.Ref
 
-	var sourceUrl string = shared.InjectVersionIntoPath(request.Source.URI, version, "<version>")
-	URI, err := url.Parse(sourceUrl)
+	var SourceURL string = shared.InjectVersionIntoPath(request.Source.URI, version, "<version>")
+	URI, err := url.Parse(SourceURL)
 	if err != nil {
```
Comment (EugenMayer) [inline src/github.com/eugenmayer/concourse-static-resource/in/in_command.go:40]: uppercase?

File: src/github.com/eugenmayer/concourse-static-resource/model/model.go [hunk 2]
```diff
@@ -5,10 +5,9 @@ type CheckRequest struct {
...
 
 type InRequest struct {
-	Source  Source  `json:"source"`
-	Version Version `json:"version"`
-	Params InParams `json:"params"`
-
+	Source  Source   `json:"source"`
+	Version Version  `json:"version"`
+	Params  InParams `json:"params"`
 }
```
Comment (EugenMayer) [inline src/github.com/eugenmayer/concourse-static-resource/model/model.go:11]: lets remove this nasty newline too in this go, thanks!

File: src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go [hunk 3]
```diff
@@ -1,21 +1,20 @@
 package shared
 
 import (
-	"github.com/eugenmayer/concourse-static-resource/model"
 	"fmt"
+
+	"github.com/eugenmayer/concourse-static-resource/model"
 )
```
Comment (EugenMayer) [inline src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go:7]: if yo can, remove those newlines too :)

**Review by EugenMayer — COMMENTED** [review]
just minor changes and we are already good to go. Thanks for the effort!

File: src/github.com/eugenmayer/concourse-static-resource/in/in_command.go (hunk 1 above)
Comment (blockchainjamie) [reply src/github.com/eugenmayer/concourse-static-resource/in/in_command.go:40]: Was flagged by `golint` inline with convention(s) defined here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go (hunk 3 above)
Comment (blockchainjamie) [reply src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go:7]: Another thing that was flagged: https://github.com/golang/go/wiki/CodeReviewComments#imports

Can remove if you want though?

Comment (EugenMayer) [reply src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go:7]: sure!

File: src/github.com/eugenmayer/concourse-static-resource/in/in_command.go (hunk 1 above)
Comment (EugenMayer) [reply src/github.com/eugenmayer/concourse-static-resource/in/in_command.go:40]: Yes, URL should be upper case, but not Source, so we should have `sourceURL`

Comment (blockchainjamie) [reply src/github.com/eugenmayer/concourse-static-resource/in/in_command.go:40]: You're right, thanks. Fixed.

**Comment by blockchainjamie** [comment]
Let me know if there's anything else.

**Comment by EugenMayer** [comment]
Thank you once again @blockchainjamie ! LGTM

**Pull request merged**